tests/deprecation/test_testcases.py

```diff
@@ -0,0 +1,37 @@
+from unittest.mock import patch
+
+from django.conf import (
+    ASSERT_FORM_SET_ERROR_RENAMED_MSG,
+    ASSERT_QUERY_SET_EQUAL_RENAMED_MSG,
+)
+from django.test import SimpleTestCase, TransactionTestCase, ignore_warnings
+from django.utils.deprecation import RemovedInDjango51Warning
+
+
+class SimpleTestCaseDeprecationTests(SimpleTestCase):
+    def test_rename_assertFormsetError_deprecation(self):
+        msg = ASSERT_FORM_SET_ERROR_RENAMED_MSG
+        with self.assertRaisesMessage(RemovedInDjango51Warning, msg):
+            self.assertFormsetError()
+
+    @ignore_warnings(category=RemovedInDjango51Warning)
+    def test_rename_assertFormsetError_calls_assertFormSetError(self):
+        with patch.object(self, "assertFormSetError") as assertFormSetError:
+            self.assertFormsetError(1, 2, a=3, b=4)
+        assertFormSetError.assert_called_once_with(1, 2, a=3, b=4)
+
+
+class TransactionTestCaseDeprecationTests(TransactionTestCase):
+
+    available_apps = ()
+
+    def test_rename_assertQuerysetEqual_deprecation(self):
+        msg = ASSERT_QUERY_SET_EQUAL_RENAMED_MSG
+        with self.assertRaisesMessage(RemovedInDjango51Warning, msg):
+            self.assertQuerysetEqual()
+
+    @ignore_warnings(category=RemovedInDjango51Warning)
+    def test_rename_assertQuerysetEqual_calls_assertQuerySetEqual(self):
+        with patch.object(self, "assertQuerySetEqual") as assertQuerySetEqual:
+            self.assertQuerysetEqual(1, 2, a=3, b=4)
+        assertQuerySetEqual.assert_called_once_with(1, 2, a=3, b=4)
```

Comment:
Using `mock` in unnecessary. It's enough to check that `assertQuerysetEqual()` work.

Reply:
Fixed in https://github.com/django/django/pull/16110/commits/4f7b6a950c9d5de6d4c8817925f12571589bf796.